journalpump/journalpump.py

```diff
@@ -675,6 +687,24 @@ def _truncate_long_message(self, json_entry):
         }
         return json.dumps(entry, default=default_json_serialization).encode("utf8")
 
+    def _apply_secret_filters(self, data):
+        changed = False
+        for idx, _ in enumerate(self.reader.secret_filters):
```

Comment:
Same here, preference for `for secret_filter in self.reader.secret_filters`, as `idx` is only being used to access that list and not a correlated list.